searx/static/themes/simple/package.json

```diff
@@ -32,8 +32,7 @@
     "eslint": "grunt eslint",
     "eslint-fix": "grunt eslint --fix",
     "watch": "grunt watch",
-    "webfont": "grunt webfont",
-    "clean": "rm -Rf node_modules package-lock.json ion.less",
+    "clean": "rm -Rf node_modules package-lock.json",
```

Comment:
Does this `ion.less` not exists (anymore) / is the `rm ion.less` a leftover from past?

Reply:
`ion.less` will used to create the webfont : https://github.com/searx/searx/blob/master/searx/static/themes/simple/less/ion.less